micromamba/tests/test_install.py

```diff
@@ -614,17 +614,6 @@ def test_no_reinstall(self, existing_cache):
         reinstall_res = helpers.install("xtensor", "--json")
         assert "actions" not in reinstall_res
 
-    @pytest.mark.skipif(
-        sys.platform == "win32",
-        reason="Set info (e.g 'name') is wrong (to be fixed)",
-    )
-    def test_install_local_package(self):
-        """Attempts to install a .tar.bz2 package from a local directory."""
-        file_path = Path(__file__).parent / "data" / "cph_test_data-0.0.1-0.tar.bz2"
-
-        res = helpers.install(f"file://{file_path}", "--json", default_channel=False)
```

Comment:
This is the wrong way of testing, we should only provide the path and `abs_path_to_url` will take care of converting it to a valid file URL format.